config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1alpha3.yaml

```diff
@@ -87,9 +89,9 @@ presubmits:
           - ./scripts/ci-e2e.sh
         env:
           - name: GINKGO_FOCUS
-            value: "Workload cluster creation"
+            value: "Optional cluster flavor test"
```

Comment:
unless we're planning on backporting https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2100 (which would probably be a merge conflict nightmare) we shouldn't touch the older release branches)

For v1beta1, we can either update it now and backport, or update the test whenever we switch the test release branch to 1.2

Reply:
I actually assumed we would do a cherry-pick exercise. :)

But yeah, let's be practical. At what point does it benefit the capz testing efforts to make this change?

Reply:
I think we should merge this and then cherry-pick this into v1beta1 now:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2100

WDYT?

If that plan works, I'll revert the changes to the v1alpha3 and v1alpha4 configs.

Reply:
only making these changes in v1beta1 presubmits (which means we'll cherry-pick https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2100 into the relevant v1beta1 branch(es) in capz)